docs/faq/known_issues.md

```diff
@@ -61,6 +61,54 @@ sudo cp /opt/ros/humble/share/pcl_ros/cmake/export_pcl_rosExport.cmake /opt/ros/
 sudo sed -i -e 's/\/opt\/ros\/humble\/include\/rclcpp;//g' /opt/ros/humble/share/pcl_ros/cmake/export_pcl_rosExport.cmake
 ```
 
+### Warnings caused by `setup.py`
+
+- Issue
+  - The following warnings happens when building CARET
```

Comment:
I think 'occur' is more formal, while 'happen' is usually used in more informal situations.
If you want to use 'happen', you should remove the 's' because 'warnings' is plural.
```suggestion
  - The following warnings occur when building CARET
```
or
```suggestion
  - The following warnings happen when building CARET
```

Reply:
Fixed it in [f251404](https://github.com/tier4/CARET_doc/pull/213/commits/f2514040d975aee375f4d8dd7c746d8c0756ed5b).